game-core/src/main/java/games/strategy/triplea/TripleA.java

```diff
@@ -98,7 +83,7 @@ public void startGame(
 
   @Override
   public Class<? extends IRemote> getRemotePlayerType() {
-    return ITripleAPlayer.class;
```

Comment:
:+1:   Good to see ITripleAPlayer be consolidated and the hierarchy collapsed. It was a useful abstraction when there were multiple game types, now that things are focused it is past time for it to be simplified.